bin/corpus

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+require 'etc'
 require 'mutant'
 require 'optparse'
+require 'pathname'
 require 'unparser'
 
+Thread.abort_on_exception = true
+Thread.report_on_exception = true
```

Comment:
__comment:__ FYI `Thread.report_on_exception` has been true by default since Ruby 2.5.

Also, It probably doesn't matter in this case since this it the top-level script, but it is possible to set these attributes at the Thread object level so you don't have to worry about doing it globally.